bee-api/bee-rest-api/src/types/body.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 IOTA Stiftung
```

Comment:
Can you replace all headers with `// Copyright 2020-2021 IOTA Stiftung` please ?